docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+            },
+            can_read: { directly_related_user_types: [] },
+          },
+        },
+      },
+    ],
+    schema_version: '1.1',
+  }}
+/>
+
+You can identify which usersets you need to add by looking at tuples in your store that have the following structure:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'group:sales#member',
+      relation: 'viewer',
+      object: 'folder:sales',
+      _description: 'Members of the "sales" group are viewers of the "sales" folder',
+    },
+  ]}
+/>
+
+If you find a tuple like that, you’ll need to add `group#member` in the list of types allowed in the `viewer` relation of the `folder` type.
+
+## Public Access
+
+When using version 1.0, you can indicate public access to specific objects by specifying a wildcard user in a relationship to any object, e.g.:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: '*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All users are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+When you write the tuple above, all users are granted with the “viewer” relationship for the “pricing" document. You can write those kinds of tuples for any relation that is <ProductConcept section="what-are-direct-and-implied-relationships" linkName="directly assignable" /> in the model.
+
+In version 1.1 we want to be more explicit about the tuples you can write, so you’ll need to declare in the DSL which relations allow wildcards and for which object types. If we want to let any object of type “user” to be a viewer of a specific document, we’ll need to explicitly define it.
+
+<AuthzModelSnippetViewer
+  configuration={{
+    type_definitions: [
+      { type: 'user', relations: {} },
+      {
+        type: 'document',
+        relations: { viewer: { this: {} } },
+        metadata: {
+          relations: {
+            viewer: {
+              directly_related_user_types: [{ type: 'user' }, { type: 'user', wildcard: {} }],
+            },
+          },
+        },
+      },
+    ],
+    schema_version: '1.1',
+  }}
+/>
+
+You’ll need to specify `user:*` as the user value in the tuple to enable this:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All objects of type "user" are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+Being explicit about the wildcard type restrictions also lets you model scenarios like “all employees can see this document, but not all external users”, “all user accounts can access this document, but not service/machine-to-machine accounts”.
+
+This change implies that you’ll need to change your code to write tuples with this new syntax, and that you’ll need to migrate existing tuples to use the new format.
+
+You might have 3 kinds of tuples in your model that use “\*”, with different migration strategies:
```

Comment:
Can we make more consistent use of `*` vs "*" ? I think it reads easier to the developer if it's `*` or `"*"` as it stands out. Otherwise it just seems like a lot of text I don't want to read through 🤣